augly/image/utils/utils.py

```diff
@@ -30,7 +30,7 @@ def validate_and_load_image(image: Union[str, Image.Image]) -> Image.Image:
     return image
 
 
-def ret_and_save_image(image: Image.Image, output_path: Optional[str]) -> Image.Image:
+def ret_and_save_image(image: Image.Image, output_path: Optional[str], src_mode: Optional[Image.Image]) -> Image.Image:
```

Comment:
```suggestion
def ret_and_save_image(
    image: Image.Image, output_path: Optional[str], src_mode: str
) -> Image.Image:
```

Make sure no line has >90 columns; also `src_mode` should be a string -- e.g. "RGBA", "RGB", or "L" :)

Reply:
Also, now that you've added `src_mode` as a param to this function, you should use it! For example you could change line 41 to:
```python4
return image.convert(src_mode)
```

Reply:
Will the `src_mode` be optional?